### ISSUE TITLE
Authorship in editor displayed incorrectly (as "false") after update

### ISSUE THREAD

**Issue by dcht00** [ether/etherpad-lite#5401]
_Checking json from server, it's already coming in as "false"._

2 separate instances exhibit same behaviour:
* editor shows author name as **_"false"_** & color as blank
* changing name&color works, but upon refresh, it's again same as above

(However concurrent users see others' name&colors fine, and it persists fine too). 

In logs, when opening editor:
`[WARN] message - Dropped message, USERINFO_UPDATE Color is malformed.[object Object]
`

I can provide a public instance example if needed.
 
____________________

Etherpad version is 1.9.0 (5d3c34a)
npm: 6.14.16
Node: v12.22.10
Plugins: None
Clients: latest Firefox @ Ubuntu

**Comment by dcht00** [comment]
Update:
* I upgraded node+npm on system
npm: 8.5.0
node: v14.19.0
* recloned one of the etherpad instances on master branch.
Etherpad version is 1.8.16 (142a47c)

That one now works fine, while the ones on 5d3c34a (develop branch) don't (might be connected to sessions, not version, though).

**Comment by g4rf** [comment]
It's the same with our public instance though it shows `true`. The error log says:
```
[2022-03-22 14:34:49.559] [ERROR] socket.io - Error handling pad message from ****: Error: COLLABROOM: USERINFO_UPDATE: malformed color: true
    at Object.exports.handleMessage (/home/etherpad/etherpad-lite/src/node/handler/PadMessageHandler.js:363:11)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async /home/etherpad/etherpad-lite/src/node/handler/SocketIORouter.js:76:14
```
Checking out version **1.8.16** [`142a47c`](https://github.com/ether/etherpad-lite/commit/142a47cbbc40d93513dd9ef1efe35c3e0f9efda6) get it working again:
```
git checkout 142a47c
```

**Comment by squiddle** [comment]
Check your `padOptions` in the settings.json/credentials.json

Maybe you have userName and userColor set to false. whereas the default padOptions in Settings.js indicate it should be null.

i had the same mistake and the settings.template was fixed in 8c857a85ac9af08ebec1c94031ca7c54bed34c38
